test/cmake/seqan3_require_test.cmake

```diff
@@ -31,10 +29,7 @@ macro (seqan3_require_test)
         gtest_project
         PREFIX gtest_project
         GIT_REPOSITORY "https://github.com/google/googletest.git"
-        # we currently have warnings that were introduced in
-        # 03867b5389516a0f185af52672cf5472fa0c159c, which are still available
-        # in "release-1.8.1", see https://github.com/google/googletest/issues/1419
```

Comment:
I suppose you deleted these lines because we no longer have these warnings?
I saw that the googletest issue was closed.

Reply:
Yes, this is left-over.